conda-forge.yml

```diff
@@ -1,7 +1,3 @@
-build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
-  osx_arm64: osx_64
```

Comment:
Also we should keep this line as we need it for macOS ARM support

Reply:
OK. I've also added notes to explain why the build platform is manually specified in each case. Removed the linux ARM specification because it is not needed. If at some point, native builds of linux ARM become the default, I assume y'all will want to switch to that.